rudalle/pipelines.py

```diff
@@ -60,11 +60,11 @@ def generate_images(text, tokenizer, dalle, vae, top_k, top_p, images_num, image
     return pil_images, scores
 
 
-def super_resolution(pil_images, realesrgan):
+def super_resolution(pil_images, realesrgan, batch_size=4):
     result = []
     for pil_image in pil_images:
         with torch.no_grad():
-            sr_image = realesrgan.predict(np.array(pil_image))
+            sr_image = realesrgan.predict(np.array(pil_image), batch_size=batch_size)
```

Comment:
here used only one pil_image, so batch_size = 1 always

Reply:
The image is split into multiple patches. On line 49, `batch_size` determines how many patches are iterated at a time.

https://github.com/sberbank-ai/ru-dalle/blob/9ec451bc127a3f6413673ce861df67f239c28778/rudalle/realesrgan/model.py#L40-L50

Reply:
yes, exactly! thank you